File: 1999/Fase 1/r3_cola.py
entrada = input()
if entrada == '0': exit()
k, a, b, c, d = [float(i) for i in entrada.split()]

trem = round(a + b*k, 2)
caminhao = round(c + d*k, 2)

if abs(trem - caminhao) < 1 or trem < caminhao: 
    print('T')
else:
    print('C')

File: 1999/Fase 1/r3_cola.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int k;
    cin >> k;
    if (k == '0') {return 0;}
    float a, b, c, d;
    cin >> a >> b >> c >> d;

    float trem = round((a + b*k) * 100) / 100;
    float caminhao = round((c + d*k) * 100) / 100;
    cout << k << ' ' << a << ' ' << b << ' ' << c << ' ' << d << ' ' << trem << ' ' << caminhao << endl;
    if (fabs(trem - caminhao) < 1 or trem < caminhao) {
        cout << 'T' << endl;
    } else {
        cout << 'C' << endl;
    }
}